test/parallel/test-eventtarget.js

```diff
@@ -12,7 +12,7 @@ const {
   ok,
   deepStrictEqual,
   strictEqual,
-  throws,
+  throws
```

Comment:
nit: this change seems unnecessary
```suggestion
  throws,
```